packages/uui-boolean-input/lib/uui-boolean-input.element.ts

```diff
@@ -142,6 +142,7 @@ export abstract class UUIBooleanInputElement extends FormControlMixin(
   }
 
   private _onKeypress(e: KeyboardEvent): void {
+    e.stopPropagation();
```

Comment:
We are not replacing this event, merely listening to it. So no need to stop propagation here.